### ISSUE TITLE
bower.json misconfigured

### ISSUE THREAD

**Issue by mastermind1981** [kuhnza/angular-google-places-autocomplete#25]
Hi,

First of all, thank you for your great job.

There is a problem in your bower.json : first, you should only expose min or src files they are both exposed as main files, second you should remove the slash in the path the actual paths are 

```
     "/src/autocomplete.js",
      "/src/autocomplete.css",
      "/dist/autocomplete.min.js",
      "/dist/autocomplete.min.css"
```

Thank you in advance

**Comment by SeanHub** [comment]
+1

Using the Grunt task Wiredep, all four are automatically pulled in. There is only need for the .min files.

**Comment by rallin** [comment]
Fixed with 8ee626a
